src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/standard/PanelMembersStatusReportITest.java

```diff
@@ -0,0 +1,265 @@
+package uk.gov.hmcts.juror.api.moj.report.standard;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.test.context.jdbc.Sql;
+import uk.gov.hmcts.juror.api.TestConstants;
+import uk.gov.hmcts.juror.api.moj.controller.reports.request.StandardReportRequest;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.AbstractReportResponse;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.StandardReportResponse;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.StandardTableData;
+import uk.gov.hmcts.juror.api.moj.report.AbstractStandardReportControllerITest;
+import uk.gov.hmcts.juror.api.moj.report.ReportHashMap;
+import uk.gov.hmcts.juror.api.moj.report.ReportLinkedMap;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@Sql({
+    "/db/truncate.sql",
+    "/db/mod/truncate.sql",
+    "/db/administration/createUsers.sql",
+    "/db/mod/reports/PanelMembersStatusReportITest_Typical.sql"
+})
+@SuppressWarnings({"PMD.LawOfDemeter", "PMD.JUnitTestsShouldIncludeAssert"})
+class PanelMembersStatusReportITest extends AbstractStandardReportControllerITest {
+
+    @Autowired
+    public PanelMembersStatusReportITest(TestRestTemplate template) {
+        super(template, PanelMembersStatusReport.class);
+    }
+
+    @Override
+    protected String getValidJwt() {
+        return getCourtJwt("415");
+    }
+
+    @Override
+    protected StandardReportRequest getValidPayload() {
+        return addReportType(StandardReportRequest.builder()
+            .trialNumber("111111")
+            .locCode(TestConstants.VALID_COURT_LOCATION)
+            .build());
+    }
+
+    @Test
+    void positiveTypicalCourt() {
+        testBuilder()
+            .triggerValid()
+            .responseConsumer(this::verifyAndRemoveReportCreated)
+            .assertEquals(getTypicalResponse());
+    }
+
+    @Test
+    void negativeUnauthorisedBureau() {
+        testBuilder()
+            .jwt(getBureauJwt())
+            .triggerInvalid()
+            .assertMojForbiddenResponse("User not allowed to access this report");
+    }
+
+    @Test
+    void negativeMissingTrialNumber() {
+        StandardReportRequest request = getValidPayload();
+        request.setTrialNumber(null);
+
+        testBuilder()
+            .payload(addReportType(request))
+            .jwt(getValidJwt())
+            .triggerInvalid()
+            .assertInvalidPathParam("trialNumber: must not be blank");
+    }
+
+    @Test
+    @SuppressWarnings({"CheckStyle.VariableDeclarationUsageDistance"})
```

Comment:
Can this be removed and have the variable set to be final

Reply:
updated